README.md

```diff
@@ -83,19 +91,21 @@ var obj = {
 }
 ```
 
--- YOUR ANSWER HERE --
+es5
+no
+obj.add(x,y)
 
 ## What does the following code output to the console?
 
 ```js
 var i = 0
-for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') ) { 
+for ( console.log('Bob');
+(console.log('Curly') === undefined); 
+console.log('Mo') ) { 
   console.log(++i)
   if (i >= 2)
     break
 }
 console.log(i)
 ```
-
--- YOUR ANSWER HERE --
-
+1
```

Comment:
The answer here is:

Bob
Curly
1
Mo
Curly
2
2

Please try playing with for loops the way Carlos did in class